### ISSUE TITLE
Add Image alt text and Caption in New Post

### ISSUE THREAD

**Issue by CharanjeetE** [mageplaza/magento-2-blog#154]
Need new functionality to add alt and caption to image in admin

### Preconditions

1. Magento: 2
2. PHP: 
3. MySQL: 
4. 

### Steps to reproduce

1. 
2. 
3. 

### Expected result

1. There should be Image alt text and Caption input fields below image upload button in megaplaza>manage posts>add new posts

### Actual result

1. [Screenshot, logs]

Please provide me some help to create new functionality: Image alt text and Caption input fields below image upload button in megaplaza>manage posts>add new posts.I found mageplaza/magento-2-blog-extension/Block/Adminhtml/Post/Edit/Tab/post.php file and added code : 
        $fieldset->addField('image', \Mageplaza\Core\Block\Adminhtml\Renderer\Image::class, [
                'name' => 'image',
                'label' => __('Image'),
                'title' => __('Image'),
                'path' => $this->imageHelper->getBaseMediaPath(Image::TEMPLATE_MEDIA_TYPE_POST)
            ]
        );
        
        //may be alt text
        $fieldset->addField('alttext', 'text', [
            'name'     => 'alttext',
            'label'    => __('Alt Text'),
            'title'    => __('Alt text'),
            'required' => false
        ]);
        $fieldset->addField('caption', 'text', [
            'name'     => 'caption',
            'label'    => __('Caption'),
            'title'    => __('Caption'),
            'required' => false
        ]);
        //end may be
But it doesn't effect in admin.May be there is another file needs to be change but which file i don't know.
Please help me or let me know what I am missing.

**Comment by thaimageplaza** [comment]
Hi,
After you change in Mageplaza\Blog\Block\Adminhtml\Post\Edit\Tab\Post.php file, please clear all cache. 
Any changes you make will be applied.
Please check again. Thank you!

**Comment by CharanjeetE** [comment]
Thanks @thaimageplaza I tried a lot but there is not any effect on frontend.Extension is installed through composer. Directory structure is app\code\Mageplaza\magento-2-blog-extension\Block\Adminhtml\Post\Edit\Tab\Post.php and also Vendor\Mageplaza\magento-2-blog-extension\Block\Adminhtml\Post\Edit\Tab\Post.php

**Comment by CharanjeetE** [comment]
Please check out what i am missing
Thanks in advance.

**Comment by thaimageplaza** [comment]
Hi,
If you want it to take effect on the frontend, it'd go through a lot of steps more.
I'd help you done this ideal customization.
please contact me at https://mageplaza.freshdesk.com/support/home.
I will continue to help you get it done.
Thank you!

**Comment by CharanjeetE** [comment]
Thanks @thaimageplaza for you valuable support i will continue with you within one or two days.
Also have opened a ticket #12423 Add Image alt text and Caption in New Post
on https://mageplaza.freshdesk.com/support/tickets/12423
